config/feature_flag.yml

```diff
@@ -0,0 +1,11 @@
+local:
```

Comment:
What does the feature_flag do?

Reply:
@daryanypl  this is the feature flag added ts.data.from.es.enabled =true. If it is true getting data from elastic search document. If it is false getting data from database